runtime/kusama/src/lib.rs

```diff
@@ -889,6 +889,222 @@ impl pallet_proxy::Trait for Runtime {
 	type AnnouncementDepositFactor = AnnouncementDepositFactor;
 }
 
+pub struct FixCouncilHistoricalVotes;
+impl frame_support::traits::OnRuntimeUpgrade for FixCouncilHistoricalVotes {
+	fn on_runtime_upgrade() -> frame_support::weights::Weight {
+		use frame_support::traits::ReservableCurrency;
+		let mut failure: Balance = 0;
+		// https://github.com/paritytech/polkadot/pull/1252/files#diff-cba4e599a9fdd88fe8d33b5ed913958d63f844186b53c5cbe9bc73a2e2944857R22
+		let old_bond = 5 * CENTS * 6;
```

Comment:
will we have integer rounding errors here? e.g. `100 / 6 * 6 != 100`

Reply:
Indeed, I think it is better to hardcode the exact value. These few lines are mostly for demonstration of what should happen.

Reply:
```
	#[test]
	fn what_is_dollar() {
		use crate::currency::{DOTS, DOLLARS};
		println!("{:?} / 6 = {:?}", DOTS, DOLLARS);
		println!("Deposit Before: {:?}", 5 * DOTS);
		println!("Deposit After: {:?}", 5 * DOLLARS);
		assert!(false);
	}
```

```
---- constants::tests::what_is_dollar stdout ----
1000000000000 / 6 = 166666666666
Deposit Before: 5000000000000
Deposit After: 833333333330
thread 'constants::tests::what_is_dollar' panicked at 'assertion failed: false', runtime/kusama/src/constants.rs:128:9
```